### ISSUE TITLE
bug in model download script

### ISSUE THREAD

**Issue by Enzo-Kerkhof** [Douwe-Spaanderman/InteractiveNet#5]
Congratulations on the work Douwe!

**Describe the bug**
While trying to download all the models using the build in feature I get an error.

**To Reproduce**
1. Run `interactivenet_download_model -t all`
2. Get:
```
(interactivenet) C:\Users\e.kerkhof\Local_Code\InteractiveNet>interactivenet_download_model -t all


Downloading and Installing Task800_WORC_MRI
Downloading and Installing Task800_WORC_MRI
Downloading files
Traceback (most recent call last):
  File "C:\Users\e.kerkhof\.conda\envs\interactivenet\lib\runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "C:\Users\e.kerkhof\.conda\envs\interactivenet\lib\runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "C:\Users\e.kerkhof\.conda\envs\interactivenet\Scripts\interactivenet_download_model.exe\__main__.py", line 7, in <module>
  File "C:\Users\e.kerkhof\.conda\envs\interactivenet\lib\site-packages\interactivenet\deploy\download_model.py", line 70, in download_and_install_model
    download_install(task=model, models=models)
  File "C:\Users\e.kerkhof\.conda\envs\interactivenet\lib\site-packages\interactivenet\deploy\download_model.py", line 88, in download_install
    with open(str(local), 'wb') as f:
OSError: [Errno 22] Invalid argument: 'C:\\Users\\e.kerkhof\\Local_Code\\InteractiveNet\\interactivenet_results\\models\\Task800_WORC_MRI.zip?download=1'
```

**Expected behavior**
Downloading of the pretrained models to the results folder.

**Environment (please complete the following information):**
 - OS Windows 10
 - Python version 3.10.15
 - MONAI version 1.2.0
 - CUDA/cuDNN version 12.4
 - GPU models and configuration 4070 SUPER

**Additional context**
Provided pull request with proposed solution.
#6

**Comment by Douwe-Spaanderman** [comment]
Hi Enzo!

Thanks for fixing this, it seems like a Windows-related error. The code is currently untested for Windows, so if you find more errors, please share them with me, and feel free to fix them like you did here!

Thanks again.

**Comment by Enzo-Kerkhof** [comment]
Hi Douwe,

Thanks for the quick response! And for the merge.

> The code is currently untested for Windows

Yes as I read in the readme, well I suppose I tested it now and it works! With the env below it runs fine:

**Environment:**
- OS Windows 10
- Python version 3.10.15
- MONAI version 1.0.0
- CUDA/cuDNN version 12.4
- GPU models and configuration 4070 SUPER

I only had to do the step mentioned here #4 to make it work, after running the pip installs in order of your readme's.

The last minor things I found where Monai label issues I believe, as if you have a scene with multiple volumes you cannot select the source volume yourself sadly. Also if there is a volume with an attached transform (e.g. MRI to CT registration) the inference fails. But I understand it might not be worth the effort to implement this.

Thanks again for all the work!
